lib/kitchen/directions/bake_chapter_review_exercises/direct_review_exercises.xhtml.erb

```diff
@@ -0,0 +1,10 @@
+<div class="os-eoc os-<%=@klass%>-container" data-type="composite-page" data-uuid-key=".<%=@klass%>">
+  <h2 data-type="document-title">
+    <span class="os-text"><%= @title %></span>
+  </h2>
+  <div data-type="metadata" style="display: none;">
+    <h1 data-type="document-title" itemprop="name"><%= @title %></h1>
```

Comment:
Will this work with internationalization?

Reply:
For now I just added this line to `bake` file:

![image](https://user-images.githubusercontent.com/40228854/116098226-309ac580-a6ab-11eb-9bad-b82b2c7611b2.png)

And it's working. In the future I'm going to apply such solution: https://github.com/openstax/kitchen/issues/185.